addons/mail/static/src/core/common/thread.xml

```diff
@@ -47,7 +47,7 @@
                         thread="props.thread"
                         messageEdition="props.messageEdition"
                         isFirstMessage="msg_first"
-                        hasActions="props.messageActions"
+                        hasActions="props.messageActions and !props.thread.isTransient"
```

Comment:
We want to keep possibility to have actions on some transient messages, and instead actions that should not work on transient messages should be told so.

The toggle start feature has `canToggleStar` that prevents it to work on `message.is_transient`, but the `is_transient` field is currently designed only with the temporary message in discuss.channel when post message is ongoing (i.e. no RPC returned yet).

The chat bot messages, when the thread is not persistent (i.e. when the livechat visitor hasn't interacted in the conversation yet) should flag these messages as transient too so that the message actions becomes correct. Right now these messages are not considered transient when they should.